__tests__/commands/remove.js

```diff
@@ -100,6 +100,14 @@ test.concurrent('removes a single scoped package', (): Promise<void> => {
   });
 });
 
+test.concurrent("removes a workspace's dependency", (): Promise<void> => {
+  return runRemove(['left-pad'], {}, 'workspaces-remove-workspace-dependency/workspace-child', async (config): Promise<
```

Comment:
This test probably doesn't make what you want - the test framework will copy the folder referenced as third parameter, and by doing so will "forget" that "workspace-child" is a child of a "workspaces-remove-workspace-dependency".

Reply:
Understood. How would you go about writing such a test?

Reply:
Just use runInstall using the base fixture, then manually run remove (`new Remove(...)`) using the `{cwd: "..."}` option to execute it in the workspace subfolder.